parcel/ParcelModel.jl

```diff
@@ -195,8 +198,13 @@ function run_parcel(IC, t_0, t_end, pp)
     info = "\nSize distribution: $(pp.size_distribution)\n"
     if pp.size_distribution == "Monodisperse"
         distr = Monodisperse{FT}()
+        distr_params = nothing
     elseif pp.size_distribution == "Gamma"
         distr = Gamma{FT}()
+        distr_params = nothing
+    elseif pp.size_distribution == "Lognormal"
+        distr = Lognormal{FT}()
```

Comment:
Could we instead make the sigma a parameter of `distr`? See  for example like `MohlerAF` struct has four fields that contain parameters (or any other parameter struct that we create)